rules/typescript.js

```diff
@@ -47,11 +51,16 @@ module.exports = {
       'error',
       { allowConstantLoopConditions: false, ignoreRhs: true },
     ],
-    '@typescript-eslint/no-untyped-public-signature': 'error',
+    /**
+     * This rule has been deprecated and replaced by
+     * explicit-module-boundary-types as of v2.17.0.
+     */
+    '@typescript-eslint/no-untyped-public-signature': 'off',
     '@typescript-eslint/no-unused-vars': [
       'error',
       { ignoreRestSiblings: true },
     ],
+    '@typescript-eslint/prefer-as-const': 'error',
     '@typescript-eslint/prefer-optional-chain': ['error'],
```

Comment:
This is a default, no need to add it explicitly - it's not done for other rules, so I'd keep it consistent: https://github.com/ridedott/eslint-config/blob/58780c910e7022e1b31c7f11adae911d81f4d9aa/index.js#L4

Reply:
fixed with https://github.com/ridedott/eslint-config/pull/314/commits/8ad156eb1bfc14ba3f60a7cd101ef0207f4422f6